src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -62,6 +63,10 @@
         <xsd:attribute name="path" type="xsd:string" />
     </xsd:complexType>
 
+    <xsd:complexType name="links">
+        <xsd:attribute name="enabled" type="xsd:boolean" />
```

Comment:
All others similar parameters use `xsd:boolean`. If we want to change this behavior, we should do it in another PR for all `enabled` parameters.